WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModel.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import Yosemite
+import protocol Storage.StorageManagerType
+
+/// View Model logic for the Bulk Variations Update
+final class BulkUpdateViewModel {
+
+    /// Represents possible states for syncing product variations.
+    enum SyncState: Equatable {
+        case syncing
+        case syncedResults
+        case syncingError
+        case notStarted
+    }
```

Comment:
I think you could just call them
`syncing `, `synced`, ` error`, `unknown|uninitialized|notStarted`

Reply:
I updated them